testing/sdk_cmd.py

```diff
@@ -154,6 +147,27 @@ def run_raw_cli(cmd, print_output=True):
     return result.returncode, stdout, stderr
 
 
+
+def run_raw_cli(cmd, print_output=True):
+    """Runs the command with `dcos` as the prefix to the shell command
+    and returns the resulting output (stdout separated from stderr by a newline).
+
+    eg. `cmd`= "package install <package-name>" results in:
+    $ dcos package install <package-name>
+    """
+    dcos_cmd = "dcos {}".format(cmd)
+    log.info("(CLI) {}".format(dcos_cmd))
+    return run_command(dcos_cmd, print_output)
+
+
+def dcos_version() -> str:
```

Comment:
The other DC/OS version related functions are in `sdk_utils`. Is the only reason we have it here the circular dependency that would result?

If we do decide to keep it here, we could at least add a `lrucache` decorator as we wouldn't expect this to change at all.

Reply:
I did not see the dcos version utils in `sdk_utils.py`. Fixed this now.